src/client/app/gear-availability-report/gear-availability.controller.js

```diff
@@ -38,12 +38,19 @@
     vm.openStartPicker = openStartPicker;
     vm.openEndPicker = openEndPicker;
     vm.validDates = validDates;
+    vm.authorized = false;
     activate();
     
     /////////////////////
 
     function activate() {
-      logger.info("Activated Create Request");
+      var profile = ProfileAccess.getProfile();
+      if(profile) {
+        vm.authorized = profile.role_id == 3;

```

Comment:
Shouldn't we really just hide this tab from the Nav Bar at the top if the users authentication isn't correct?